package.json

```diff
@@ -16,16 +16,27 @@
     "coffeelint": "^1.9.7"
   },
   "configSchema": {
+    "autocompleteCharacters": {
+      "description": "Autocompleted characters treated as matching pairs, such as `()`, and `{}`.",
+      "type": "array",
+      "default": ["()","[]","{}","\"\"","''","``","“”","‘’","«»","‹›"],
+      "items": {
+        "type": "string"
+      }
+    },
+    "pairsWithExtraNewline": {
+      "description": "Automatically add a newline between the pair when enter is pressed.",
+      "type": "array",
+      "default": ["()","[]","{}"],
+      "items": {
+        "type": "string"
+      }
+    },
     "autocompleteBrackets": {
       "type": "boolean",
       "default": true,
       "description": "Autocomplete bracket and quote characters, such as `(` and `)`, and `\"`."
```

Comment:
> Autocomplete the characters defined in the `autocompleteBrackets` setting

This setting name is a bit misleading now, but we can't change it without breaking existing configs.  Oh well.